products/jbrowse-cli/src/commands/admin-server.test.ts

```diff
@@ -0,0 +1,216 @@
+/**
+ * @jest-environment node
+ */
+
+import fs from 'fs'
+import * as path from 'path'
+import fetch from 'node-fetch'
+import { Context } from 'vm'
```

Comment:
This is probably not the context you want

Reply:
will remove and define type inline like below